prow/cmd/build/controller_test.go

```diff
@@ -1212,10 +1224,11 @@ func TestDecorateBuild_Timeout(t *testing.T) {
 		expected         *time.Duration
 	}{
 		{
-			name: "do not change timeout when decorated unset",
+			name:     "set the default timeout when decoration is unset",
+			expected: &defaultTimeout,
 		},
 		{
-			name:         "do not change set timeout (decoration unset)",
```

Comment:
Why is this change necessary?

Reply:
It was unintentional. Restored the previous test name.